packages/graphql/tests/e2e/subscriptions/filtering/create.e2e.test.ts

```diff
@@ -147,6 +147,232 @@ describe("Create Subscription with optional filters valid for all types", () =>
         expect(wsClient.events).toEqual([]);
     });
 
+    test("create subscription with where OR", async () => {
```

Comment:
I may have missed some test cases, but I would like to make sure the following cases are tested:
```
(where: { OR: [{ title: "movie1"}] })    // A single element inside OR
(where: {OR: [{ title: "movie1"}, AND: [{title: "movie2" } ,  {title: "Movie3"}] // Nested conditionals

```

Reply:
on it

Reply:
done